Add grid version validation (#1018)
demisto/demisto-sdk: CHANGELOG.md

* Fixed an issue where an irrelevant id_set validation ran in the **validate** command when using the *--id-set* flag.
* Fixed an issue were **generate-docs** command has failed if a command did not exist in commands permissions file.
* Improved a **validate** command message for missing release notes of api module dependencies.

> * Added a validation for invalid fromVersion field in indicators of Grid type in **validate** command.

Reviewer: ```suggestion
* Added a validation for the version of incident field of type grid.
```
